src/views/Wallet/components/LeftNavigation/components/CoinMenu/index.js

```diff
@@ -44,15 +48,14 @@ class CoinMenu extends Component {
                 <Divider
                     textLeft="Other coins"
                     textRight="(You will be redirected)"
-                    borderTop
-                    borderBottom
+                    hasBorder
                 />
                 {coins.map(coin => (
-                    <a key={coin.url} href={coin.url}>
+                    <a key={this.getCoinUrl(coin.id)} href={this.getCoinUrl(coin.id)}>
```

Comment:
not sure if it is good idea, what if there will be some coin which is not on `https://wallet.trezor.io/#/coin/${coinId}` (example: myetherwallet, nanowallet, etc)